src/main/java/com/epam/test_generator/entities/SuitResult.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.test_generator.entities;
+
+import java.time.Duration;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+@Entity
+public class SuitResult {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private String name;
+    private long duration;
+
+    @OneToMany(cascade = {CascadeType.ALL})
+    private List<CaseResult> cases;
```

Comment:
I'd suggest naming it caseResultList or caseResults to avoid confusion between case and caseResult